extensions/extensions-helm/aissemble-airflow-chart/values.yaml

```diff
@@ -2,17 +2,7 @@
 ## CONFIG | Airflow Configs
 ########################################
 airflow:
-  airflowVersion: 2.6.2
-
-
-  # NB: 1.7.0 is the last version to include airflow support. It is being frozen there until it is removed or pulled
-  # into an extension project.  TODO in https://github.com/boozallen/aissemble/issues/227
-  defaultAirflowRepository: ghcr.io/boozallen/aissemble-airflow
-  defaultAirflowTag: 1.7.0
-
-  images:
-    airflow:
-      pullPolicy: Always
```

Comment:
These properties need to be removed from the README

Reply:
Removed.